app/assets/javascripts/covid19/src/webapp/components/app/Root.tsx

```diff
@@ -55,6 +57,8 @@ export const Root: React.FC = React.memo(() => {
                         search={search}
```

Comment:
I think the time has come: can we create a component that wraps all the proteome stuff?

Reply:
Sure!